nucleus/modelci/README.md

```diff
@@ -0,0 +1,45 @@
+# Notes on defining Evaluation Criteria for a test
```

Comment:
This is great documentation, but I think here it may not be very discoverable. Can you include the stuff under this header in the class docstring for EvaluationCriterion instead? I think users will find it more easily there.

Reply:
I think both headers can go under the class docstring for evaluation criterion. This way it will show up in our API docs.

I think users will either look at our "how to guides" or our API docs in most cases.

Reply:
+1 I think this README is fantastic and should stay. Maybe we could add a summary of this explanation to the `EvaluationCriterion` class docstring.